backend/development.ini.sample

```diff
@@ -243,37 +257,39 @@ api.key = %(basic_setup.api_key)s
 ; user.auth_token.validity = 604800
 ## user reset_password token lifetime (default to 900s -> 15 minutes)
 ; user.reset_password.token_lifetime = 900
-## Default profile of created user, valid values are users, trusted-users, administrators
+## Default profile of created user, valid values are: users, trusted-users, administrators
 ; user.default_profile = users
 
-## active or disable filter on known user, if activate user cannot see at all informations about
-## member with no common workspace. if false, it can access to these informations.
+## enable or disable filter on known user, if True, user cannot see at all informations about
+## member with no common workspace. If False, it can access to these informations.
 ## This is used by search mechanism for invitation of user in workspace.
-## default value to true.
 ; known_members.filter = True
+
 ####
 # IHM
 ####
 
 ### Website config ###
 ## title of website, used in email
+; website.title = Tracim
 website.title = %(basic_setup.website_title)s
 # The following base_url is used for links and icons
 # integrated in the email notifcations, it's the frontend config.
+; website.base_url = http://localhost:6543
 website.base_url = %(basic_setup.website_base_url)s
 
-
-
 ### Frontend ###
-## if you want to use Tracim API without frontend, change this parameter to False
+## if you want to use Tracim API without frontend, change this parameter
 ; frontend.serve = True
-# You can set dist folder of Tracim frontend. by default, system
+# You can set dist folder of Tracim frontend. By default, system
 # will try to get it automatically according to Tracim repository
 # organisation.
-; frontend.dist_folder_path = /home/user/tracim/frontend/dist
+; frontend.dist_folder_path =
+# example:
+# frontend.dist_folder_path = /home/user/tracim/frontend/dist
```

Comment:
Whe backend does not use an `(here)` mechanism ?

Reply:
That's and example.